src/material-table.js

```diff
@@ -83,7 +83,11 @@ export default class MaterialTable extends React.Component {
     isInit && this.dataManager.changeOrder(defaultSortColumnIndex, defaultSortDirection);
     isInit && this.dataManager.changeSearchText(props.options.searchText || '');
     isInit && this.dataManager.changeCurrentPage(props.options.initialPage ? props.options.initialPage : 0);
-    (isInit || this.isRemoteData()) && this.dataManager.changePageSize(props.options.pageSize);
+    
+    if(isInit && this.isRemoteData()) { 
```

Comment:
I am not sure this is true condition. I know that we have to find a better solution instead of a workaround.

Reply:
it is your call, and you could find a better solution, but right now it is a big issue for people are using the table in remote mode.